crypto/rand/rand_unix.c

```diff
@@ -88,6 +86,8 @@ int syscall_random(void *buf, size_t buflen);
     || defined(OPENSSL_SYS_VMS) || defined(OPENSSL_SYS_VXWORKS) \
     || defined(OPENSSL_SYS_UEFI))
 
+static ssize_t syscall_random(void *buf, size_t buflen);
+
```

Comment:
Note: The declaration was moved out of sight of the MSVC compiler to fix this [AppVeyor build failure](https://ci.appveyor.com/project/openssl/openssl/build/master.19459/job/j7ufl0mpi9q33ugu).